policies/queries/autoscaling/no_target_groups.sql

```diff
@@ -0,0 +1,12 @@
+insert into aws_policy_results
+select :'execution_time'                                            as execution_time,
+       :'framework'                                                 as framework,
+       :'check_id'                                                  as check_id,
+       'Autoscaling groups with no target groups should be removed' as title,
+       account_id,
+       arn                                                          as resource_id,
+       case
+           when length(target_group_arns) > 0 then 'pass'
+           else 'fail'
+           end                                                      as status
+from aws_autoscaling_groups
```

Comment:
Not all autoscaling groups serve traffic behind a load balancer... For example an ASG might be required to make sure that a pool of workers pulling work off of a queue always has a certain number of instances

Reply:
I'll remove this check for now altogether